_posts/2018-07-20-exploring-state-management-in-react-container-components.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Container Components&#58; Exploring State Management in React (Part I)
+date: 2018-07-25 00:00:00 -04:00
+categories: post
+desc: This one simple trick will turn your unruly React components into the obedient angels they were meant to be!
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+>At what level of complexity will my React application require Redux?
+
+React developers have been asking this question for a long time, and answers still vary wildly. The truth is there is quite a bit you can do before needing to pull in Redux, and even then, _**Redux**_ isn't your only option! 
```

Comment:
💡 Some people feel that using 'you' separates you, the author, from you, the reader. 'We' can engage the audience more.

Personally, I'm on the fence, but I know some people feel very strongly about it...

Reply:
I think I agree with that.